### PR TITLE
Correct dependencies in agent spec file - backport to b0.71

### PR DESCRIPTION
**Pull request by ndokos**
This is a backport of #2901 to b0.71.

### CODE REVIEW

**Review by dbutenhof — DISMISSED** [review]
Looks like what we agreed. Makes me wish we had a functional test system that would show a reviewer clearly that this all works. 😆

**Comment by portante** [comment]
Set to `Draft` until #2901 is resolved.

**Review by webbnh — APPROVED** [review]
👍

**Review by portante — CHANGES_REQUESTED** [review]
This needs updating now that PR #2901 has been merged.

**Comment by dbutenhof** [comment]
Unless my Chrome is refusing to update the view (which *has* happened), you're showing merge conflicts and it's still marked Draft. ???

File: utils/rpm.mk [hunk 1]
```diff
@@ -41,7 +41,7 @@ srpm: spec patches tarball
 .PHONY: spec
 spec: rpm-dirs ${prog}.spec.j2
 	if [ -e ./seqno ] ;then expr ${seqno} + 1 > ./seqno ;fi
-	jinja2 ${prog}.spec.j2 -D version=${VERSION} -D gdist=g${sha1} -D seqno=${seqno} > ${RPMSPEC}/${prog}.spec
+	jinja2 ${prog}.spec.j2 targets.json -D version=${VERSION} -D gdist=g${sha1} -D seqno=${seqno} > ${RPMSPEC}/${prog}.spec
```
[the remark this answers is not in the document]
Comment (ndokos) [reply utils/rpm.mk:44]: OK - I'll open an issue and try to fix it (against main, not against b0.71).

**Review by npalaska — APPROVED** [review]

**Comment by ndokos** [comment]
Closing this PR in favor of #2927 which produces identical results and simplifies the implementation considerably.

**Comment by portante** [comment]
FWIW, the `Backport` label is used to mark PRs which should be back-ported, not PRs that _are_ backports.

**Pull request closed without merging**